ilert/resource_status_page.go

```diff
@@ -0,0 +1,457 @@
+package ilert
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/iLert/ilert-go/v2"
+)
+
+func resourceStatusPage() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validation.StringLenBetween(1, 255),
+			},
+			"domain": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"subdomain": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"timezone": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "Europe/Berlin",
+				ValidateFunc: validation.StringInSlice([]string{
+					"Europe/Berlin",
+					"America/New_York",
+					"America/Los_Angeles",
+					"Asia/Istanbul",
+				}, false),
```

Comment:
this is not the complete timezone list. please remove te validation or use the complete list